package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "loop-segment",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "clear": "rm -rf dist || del /s /q dist",
+    "build": "npm run clear && tsc",
+    "lint": "eslint ./ --ignore-path .gitignore",
+    "lint:fix": "npm run lint -- --fix",
+    "prepare": "husky install"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Author Should be Olive. License should probably MIT and add copyright 
`  "copyright": "Copyright 2022 Olive"`